udf-compiler/README.md

```diff
@@ -20,6 +20,7 @@ export SPARK_HOME=[your spark distribution directory]
 export JARS=[path to cudf 0.15 jar]
 
 $SPARK_HOME/bin/spark-shell \
---jars $JARS/cudf-0.15-cuda10-1.jar,udf-compiler/target/rapids-4-spark-udf_2.12-0.2.0-SNAPSHOT.jar,sql-plugin/target/rapids-4-spark-sql_2.12-0.2.0-SNAPSHOT.jar \
---conf spark.sql.extensions="com.nvidia.spark.SQLPlugin,com.nvidia.spark.udf.Plugin"
+--jars $JARS/cudf-0.15-cuda10-1.jar,dist/target/rapids-4-spark_2.12-0.2.0-SNAPSHOT.jar \
```

Comment:
Rather than replicate the startup command-line from elsewhere in the documentation (and have to remember to update it when it changes), should we just reference the existing documentation on using the plugin or just mention the config(s) specific to running it?

Reply:
Sure, it does not fix the issue with the fact that this is in 4 other places too, but at least it would be a step in the right direction.